.python-version

```diff
@@ -0,0 +1 @@
+3.6.9
```

Comment:
Are we using 3.6.9 on the scripts docker image? I see `python:3.6-slim`

Reply:
3.6-slim currently points to Python 3.6.10, unfortunately however we can't use 3.6.9 for Dependabot just yet:

https://github.com/dependabot/dependabot-core/blob/master/python/lib/dependabot/python/python_versions.rb#L12-L17